myRBM/rumination.m

```diff
@@ -1,38 +1,19 @@
 %% rumination
-%% manual manipulation of learning rate
-loadData
-
-epsilon1 = 0.001;
-epsilon2 = 0.01; % ruminatory
-
-ts = [4,3,2,19,5,9,12,18,62,8];
-
-[M1, b1, c1, errors1] = rbm(epsilon1, Ni, trImages);
-[M2, b2, c2, errors2] = rbm(epsilon2, Ni, trImages); % ruminatory
-
-for i = 1:length(ts)
-    test_input(M1, 10, c1, b1, tsImages(ts(i),:), "control");
-    test_input(M2, 10, c2, b2, tsImages(ts(i),:), "rumination");
-end
-
-v0 = rand(Ni,1);
-test_unclamped(M1, 10, c1, b1, v0);
-test_unclamped(M2, 10, c2, b2, v0);
-
 %% phase 1 - attentional breadth (trait vs. control)
 loadData
 
 nhidden = 100; % number of hidden units - modulate - see Hinton's recipe
-epsilon = 0.001;
+epsilon = 0.005; % meets Hinton's guidelines, seems to be an okay value for our learning
 maxEpochs = 4; % number of training epochs
 cdk = 1; % contrastive-divergence steps
 
-cBreadth = 0.99;
-eBreadth = 0.90;
-acdk = 100; % associative recall k-step
-rcdk = 30; % rumination k-step
-rCycles = 4;
-v0 = rand(Ni,1); % noise
+cBreadth = 0.99; % % of neurons kept on for the control case
+eBreadth = 0.90; % """ for the experimental (trait rumination case)
```

Comment:
have you played with reducing this even more? Is there any empirical evidence quantifying 'attentional breadth' that you could use to inform these %s?